### PR TITLE
User defined externals 

### PR DESCRIPTION
**Pull request by klaaz0r**

### CODE REVIEW

**Comment by EECOLOR** [comment]
You can use the link in #124 to create a test case, it contains a reference to a version of GSAP that should be whitelisted (or parsed) to prevent errors. So the configuration property should be something like `compileWithBabel`, I'm not sure what the best name would be.

**Comment by klaaz0r** [comment]
@EECOLOR I added the changes we made last friday

**Comment by EECOLOR** [comment]
The `compileWithBabel: ['gsap']` is commented out, any reason for that? A production build should of example break without this enabled.

**Comment by EECOLOR** [comment]
Also update the documentation and make sure to include a reference to: https://webpack.js.org/configuration/module/#condition

File: docs/src/content/configuration/index.raw.md [hunk 1]
```diff
@@ -79,6 +79,19 @@ module.exports = {
 }
 ```
 
+#### node externals
+
+In the case that you want to force a node_module to be parsed with Babel you can add them to
+`compileWithBabel`, The config works the same as the [`webpack configuration`](https://webpack.js.org/configuration/module/#condition).
```
Comment (EECOLOR) [inline docs/src/content/configuration/index.raw.md]: Hier moet ook een referentie naar https://www.npmjs.com/package/webpack-node-externals#optionswhitelist- bij

**Pull request merged**